docs/user/REST.rst

```diff
@@ -2,6 +2,261 @@
+            # that our predict_proba method outputs actual probabilities!
+            out = F.softmax(self.output(drop), dim=-1)
+            return out
+
+
+You can find the full contents of the ``model.py`` file in the
+``skorch/examples/rnn_classifer`` folder of Skorch's source code.
+
+Now with dataset and model in place, it's time to try Palladium out.
+You can install Palladium with ``pip install palladium``.
+
+From within the directory that contains ``model.py`` and
+``palladium-config.py`` now run the following command::
+
+  PALLADIUM_CONFIG=palladium-config.py pld-fit --evaluate
+
+You should see output similar to this::
+
+  INFO:palladium:Loading data...
+  INFO:palladium:Loading data done in 0.607 sec.
+  INFO:palladium:Fitting model...
+    epoch    train_loss    valid_acc    valid_loss     dur
+  -------  ------------  -----------  ------------  ------
+        1        0.7679       0.5008        0.7617  3.1300
+        2        0.6385       0.7100        0.5840  3.1247
+        3        0.5430       0.7438        0.5518  3.1317
+        4        0.4736       0.7480        0.5424  3.1373
+        5        0.4253       0.7448        0.5832  3.1433
+  INFO:palladium:Fitting model done in 29.060 sec.
+  DEBUG:palladium:Evaluating model on train set...
+  INFO:palladium:Train score: 0.83068
+  DEBUG:palladium:Evaluating model on train set done in 6.743 sec.
+  DEBUG:palladium:Evaluating model on test set...
+  INFO:palladium:Test score:  0.75428
+  DEBUG:palladium:Evaluating model on test set done in 6.476 sec.
+  INFO:palladium:Writing model...
+  INFO:palladium:Writing model done in 0.694 sec.
+  INFO:palladium:Wrote model with version 1.
+
+Congratulations, you've trained your first model with Palladium!
+You're ready to now serve to model on the web.  Add this piece of
+configuration to the ``palladium-config.py`` configuration file (and
+make sure it lives within the outermost brackets:
+
+.. code:: python
+
+    {
+        # ...
+
+        'predict_service': {
+            '__factory__': 'palladium.server.PredictService',
+            'mapping': [
+                ('text', 'str'),
+            ],
+            'predict_proba': True,
+            'unwrap_sample': True,
+        },
+
+        # ...
+    }
+
+With this piece of information inside the configuration, we're ready
+to launch the web server using::
+
+  PALLADIUM_CONFIG=palladium=config.py pld-devserver
+
+You can now try out the web service at this address:
+http://localhost:5000/predict?text=this+movie+was+brilliant
+
+You should see a JSON string returned that looks something like this:
+
+.. code:: json
+
+    {
+        "metadata": {"error_code": 0, "status": "OK"},
+        "result": [0.326442807912827, 0.673557221889496],
+    }
+
+The ``result`` entry has the probabilities.  Our model assigns 67%
+probability to the sentence "this movie was brilliant" to be positive.
+By the way, the Skorch tutorial itself has tips on how to improve this
+model.
```

Comment:
Maybe, as a last paragraph, you could again put a pointer to palladium and maybe add an elevator pitch for why to use it.